server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -677,6 +677,18 @@ protected Table getTableWithHeader(final RestRequest request, final PageToken pa
         );
         table.addCell("pri.search.concurrent_avg_slice_count", "default:false;text-align:right;desc:average query concurrency");
 
+        table.addCell(
+            "search.startree_query_total",
+            "sibling:pri;alias:stqc,startreeQueryTotal;default:false;text-align:right;desc:total star tree resolved queries"
```

Comment:
```
total star tree resolved queries
```
should be 
```
total star tree resolved aggregations
```

Reply:
I prefer calling it `queries` given that a search query resolved using star-tree can be purely aggregation request or it can have an optional query component as well - so a broad term `query` makes it more meaningful. 

@bharath-techie thoughts?